composer.json

```diff
@@ -76,7 +76,7 @@
         "iron-io/iron_mq": "~2.0",
         "mockery/mockery": "~0.9.2",
         "pda/pheanstalk": "~3.0",
-        "phpunit/phpunit": "~4.0",
+        "phpunit/phpunit": "~4.1",
```

Comment:
Why are you updating phpunit within this PR? If phpunit will be updated let's move this to another PR?!